kogito-editors-js/packages/boxed-expression-component/src/components/BoxedExpressionEditor/BoxedExpressionProvider.tsx

```diff
@@ -44,22 +44,24 @@ export function BoxedExpressionProvider(props: BoxedExpressionProviderProps) {
   }, [props.expressionDefinition]);
 
   return (
-    <BoxedExpressionGlobalContext.Provider
-      value={{
-        pmmlParams: props.pmmlParams,
-        supervisorHash,
-        setSupervisorHash,
-        boxedExpressionEditorRef,
-        currentlyOpenedHandlerCallback,
-        setCurrentlyOpenedHandlerCallback,
-      }}
-    >
-      <ResizerSupervisor isRunnerTable={props.isRunnerTable}>
-        <div className="boxed-expression-editor" ref={boxedExpressionEditorRef}>
-          {props.children}
-        </div>
-      </ResizerSupervisor>
-      {props.isRunnerTable === false && <CellSelectionBox />}
-    </BoxedExpressionGlobalContext.Provider>
+    <div id={"boxed-expression-editor"}>
```

Comment:
I think we should avoid using `ids` for the case where we need more than one instance of BoxedExpressionComponent in a page. Can you please change this to `className`?

Reply:
Ok, I just realized using `className` won't work too :/ The solution would be having the consumer of BoxedExpressionComponent set this `id`, delegating the responsibility of maintaining the uniqueness of this `id` to the consumer, or use a `ref` for this `div` and pass it along using props or even a context. WDYT?

Reply:
It would be a good solution. I'm going to use the` BoxedExpressionGlobalContext` to pass it down on the tree, and remove the `id`.